lib/streamlit/app_session.py

```diff
@@ -193,16 +193,16 @@ def enqueue(self, msg: ForwardMsg) -> None:
         if self._message_enqueued_callback:
             self._message_enqueued_callback()
 
-    def enqueue_exception(self, e: BaseException) -> None:
-        """Enqueue an Exception message."""
+    def handle_backmsg_exception(self, e: BaseException) -> None:
+        """Handle an Exception raised while processing a BackMsg from the browser."""
```

Comment:
I also changed the name of this function because it does quite a bit more than just "enqueue an exception".